### PR TITLE
contributing: Clarify PR workflow

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

**Comment by zachmullen** [comment]
Let's mention the "LGTM" by the reviewer as the marker that a PR is ready to be merged.

**Comment by zachmullen** [comment]
Also mention that once you merge the branch, you should also delete the topic branch (if you have write access to girder/girder).

**Comment by jcfr** [comment]
> you should also delete the topic branch

I guess for folks who have not push access to girder, the branch will be in their own fork ? 

What should be the default flow ? Push the topic to your own fork or the main girder repo ?

**Comment by zachmullen** [comment]
Main girder repo for those who have write access. Makes it easy for other devs to checkout the branch without having to add a remote of have `hub` installed.

**Comment by jeffbaumes** [comment]
Central branches also makes it easy to co-develop on branches.

**Comment by danlamanna** [comment]
Should we make a note about the WIP -> Ready for Review workflow, or is that more suited to internal collaborators?

**Comment by jcfr** [comment]
@jeffbaumes @zachmullen I just pushed an update. PTAL

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,6 +13,43 @@ We encourage a range of Pull Requests, from patches that include passing tests a
...
+Getting your contributions integrated is relatively straightforward, here
+is the checklist:
+
+* All tests pass
+* Consensus is reached. This basically mean that the reviewers add `LGTM` comments. `LGTM` is a
+shortcut for _Looks Good to Me_.
+
+Next, there are two scenarios:
+* You do NOT have push access: A girder core developer will integrate your request.
+* You have push access: Simply click on "Merge pull request" button.
```
Comment (zachmullen) [inline CONTRIBUTING.md]: Let's add

> Then click the "Delete branch" button that appears afterward.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -13,6 +13,45 @@ We encourage a range of Pull Requests, from patches that include passing tests a
...
+
+Then, click on the "Delete branch" button that appears afterward.
+
+#### Are PR tested ?
```
Comment (zachmullen) [inline CONTRIBUTING.md]: Let's change this section header to "Automatic testing of pull requests"

**Comment by jcfr** [comment]
@zachmullen Et voila

**Comment by zachmullen** [comment]
LGTM

**Pull request merged**